desktop_version/README.md

```diff
@@ -23,6 +23,11 @@ Steamworks support is included and the DLL is loaded dynamically, you do not
 need the SDK headers and there is no special Steam or non-Steam version. The
 current implementation has been tested with Steamworks SDK v1.46.
 
+If you want to build the Make and Play edition of the game, then make sure in [MakeandPlay.h](https://github.com/TerryCavanagh/VVVVVV/blob/master/desktop_version/src/MakeAndPlay.h), line 8 is uncommented like this:
+```
+#define MAKEANDPLAY
+```
+
```

Comment:
* Not a huge fan of referring to a specific line number when that line number may change in the future without automatically updating the reference to the line number.
* Not a fan of hotlinking to this very GitHub repository, rather than just a relative file path (`src/MakeAndPlay.h`).
* Why doesn't the code block have syntax highlighting?
* Please word-wrap your lines, they are too long. If you look at the rest of the file, we generally wrap at around 80 columns.

Reply:
> Why doesn't the code block have syntax highlighting?
idk